examples/basic.py

```diff
@@ -15,22 +17,28 @@ def evaluate_metric(assignments):
 
   connection = Connection(client_token=the_args.client_token, _show_deprecation_warning=False)
 
+  
+  sigopt.log_dataset("Example dataset") #Descriptor of what kind of dataset you are modeling
+  sigopt.log_metadata(key="Dataset Source", value="Example Source") #Useful for keeping track of where you got the data 
+  sigopt.log_metadata(key="Feature Pipeline Name", value="Example Pipeline") #e.g. Sklern, xgboost, etc.
+  sigopt.log_model("Example Model Technique") # What kind of learning you are attemping
   # Create an experiment with one paramter, x
-  experiment = connection.experiments().create(
+  experiment = sigopt.create_experiment(
     name="Basic Test experiment",
     project="sigopt-examples",
+    type="offline",
```

Comment:
pretty sure this is the default, in which case we prefer not to include this since the name is a bit legacy